cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -330,7 +394,7 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Sort",
       text =
-        """Sorts rows by a provided key.""".stripMargin,
+        """The `Sort` operator sorts rows by a provided key.""".stripMargin,
```

Comment:
To be able to do this, it needs to eagerly pull in all data from its source operator and keep it query state before being able to actually sort the data. This leads to added memory pressure.

Reply:
👍